README.md

```diff
@@ -11,3 +11,14 @@
 Please note that all of the code is still undocumented, and no contribution guidelines are in place.
 
 Contributions are welcome, just beware of the dragons. 🐲
+
+## Environment options
+
+The app currently supports setting a few environment settings via environment variables:
+
+- `REACT_APP_DEFAULT_ETH_NODE`: Url of the default Ethereum node to read blockchain data from (must be WebSocket protocol)
+- `REACT_APP_ENS_REGISTRY_ADDRESS`: Address of the ENS registry that APM repos were registered on
+- `REACT_APP_ETH_NETWORK_TYPE`: Either one of `rinkeby` or `local`
+- `REACT_APP_IPFS_GATEWAY`: Url of the IPFS gateway to load APM repos from
```

Comment:
If running IPFS locally I suppose the default I need to use is `http://localhost:8080` as per https://dweb-primer.ipfs.io/classical-web/lessons/local-gateway.html

Should we add another option for configuring the IPFS RPC? Where is each of them used?

Reply:
IPFS RPC is only used to fetch the artifact.json and manifest.json, AFAIK.

And yes, we probably should add something to configure it, and having a flag to switch everything from `https/wss` to `http/ws`.